provision/setup-agent.sh

```diff
@@ -54,3 +54,15 @@ else
 fi
 
 cp /vagrant/files/agent/montagu-auth /usr/bin/
+
+# Ensure the agent can ssh to support to deploy latest code - don't assume known_hosts doesn't exist
+SUPPORTHOST=$(ssh-keyscan support.montagu.dide.ic.ac.uk)
+KNOWN_HOSTS_FILE=$TEAMCITY_DIR/.ssh/known_hosts
+if [ ! -f  $KNOWN_HOSTS_FILE ]; then
```

Comment:
I think you don't need the second clause here (though it doesn't hurt) because these are all written with the idea of "run if it seems we're on a fresh machine creation".  The logic otherwise ends up a bit painful

Reply:
I just tried running this and it leaves the known host file owned by root! Please add

```
chown $TEAMCITY_USER:$TEAMCITY_GROUP $KNOWN_HOSTS_FILE
```

or move this block above the existing `chown` command (line 37 at the moment)

Reply:
Yeah, I didn't want to assume that the file did or didn't exist, just in case things got moved around again. Can take it out if that's a fair assumption though..?

Oops re file ownership!